SU-SWS/stanford_components
# Earth 302

# READY FOR REVIEW

# Summary
- Refactored filmstrip, film_card, section_tall_filmstrip

# Needed By (Date)
- before you leave?

# Urgency
- low

# Steps to Test

1. Checkout branch
2. `drush cr`
3. view patterns page to verify still _close_
note: I changed the image style on film_card to allow for tall filmstrips.  The current styles relies on the image size to be correct. I suggest we alter the template/css to allow any size image so that we can reuse this.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
